Project-I-Social-Card/social-card/src/components/FooterComponents/Footer.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import "./Footer.css";
+
+const Footer = () => (
+	<footer>
+		<icon className="comment">
```

Comment:
```suggestion
		<div className="comment">
```
icon isn't a valid tag, I would just use a div, but here is some info on creating custom elements if you would like https://html.spec.whatwg.org/multipage/custom-elements.html#custom-elements